check_repo.py

```diff
@@ -33,7 +32,18 @@ def check_repo():
     for addon_folder in toplevel_folders:
         if addon_folder[0] != '.':
             addon_path = os.path.join(repo_path, addon_folder)
-            error_counter = check_addon.start(error_counter, addon_path, config)
+            error_counter = check_addon.start(
+                error_counter, addon_path, config)
+
+    if check_addon.check_config(config, "comment_on_pull"):
+        if check_addon.comments:
+            GithubAPI().comment_on_pull(check_addon.comments)
+            GithubAPI().set_label(["Checks failed"])
+        else:
+            GithubAPI().remove_label(["Checks failed"])
+            GithubAPI().set_label(["Checks passed"])
+    else:
+        colorPrint("No config file found", "7")
```

Comment:
This might not mean that the config file is not found, it might mean that it's just set to false

Reply:
I think I should remove this because if someone is intentionally going to set config to false then we don't have to show them any message saying "Config is set to false".